test/contract-tests/tests/test/amazon/awssdk/awssdk_test.py

```diff
@@ -550,6 +550,32 @@ def test_bedrock_get_guardrail(self):
             span_name="Bedrock.GetGuardrail",
         )
 
+    def test_bedrock_runtime_invoke_model_nova(self):
+        self.do_test_requests(
+            "bedrock/invokemodel/invoke-model-nova",
+            "GET",
+            200,
+            0,
+            0,
+            rpc_service="Bedrock Runtime",
+            remote_service="AWS::BedrockRuntime",
+            remote_operation="InvokeModel",
+            remote_resource_type="AWS::Bedrock::Model",
+            remote_resource_identifier="us.amazon.nova-micro-v1:0",
+            cloudformation_primary_identifier="us.amazon.nova-micro-v1:0",
+            request_response_specific_attributes={
+                _GEN_AI_SYSTEM: "aws_bedrock",
```

Comment:
Small change. Can you update this to assert `aws.bedrock` for this new model?

We changed this recently to align better with upstream Otel semantic conventions.

Related [PR](https://github.com/aws-observability/aws-otel-dotnet-instrumentation/pull/155)

Reply:
Fixed.